src/components/TimeRelated.tsx

```diff
@@ -11,9 +11,9 @@ export interface IProps {
 }
 
 const SetHours = (timeData: string) => {
-    const currentHour = getHours(new Date());
-    const inputHours = getHours(new Date(timeData));
-    return inputHours === currentHour ? 'Сейчас' : format(new Date(timeData), 'HH:mm');
+    const currentHour = new Date();
+    const inputHours = new Date(timeData);
```

Comment:
These consts are not hours, they are instances of Date (contain not only hours), so it's better to name them as dates